frontend/src/components/pages/spells/SpellListItem.js

```diff
@@ -0,0 +1,52 @@
+import { Card, CardActions, CardContent, CardHeader, CardMedia, Grid, Typography } from "@mui/material";
+import Image from "next/image";
+
+import { faWandSparkles } from "@fortawesome/free-solid-svg-icons";
+
+import Link from '../../Link'
+import ListItemBox from '../../ui/ListItemBox'
+
+
+const SpellListItem = ({ spell }) => {
+  const { name, slug, effect, hand, incantation, category, light, creator, image, wiki } = spell.attributes;
+  
+  return (
+    <Grid item xs={12} sm={6} md={4} lg={3}>
+      <Card sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        border: '3px solid',
```

Comment:
A border color would be nice. Maybe making the color depending on the `light` field? Basically something like [this](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/src/lib/utils.js#L1).